openapi-spec/swagger.yaml

```diff
@@ -1927,6 +2057,56 @@ components:
             type: string
           example:
             VolumeId: vol-03deedaee2de0060f
+    Tier:
+      type: object
+      required:
+        - Name
+      properties:
+        Id:
+          description: 'The SSP ID'
+          type: string
+          example: 5ef1db51202b510001c3804c
```

Comment:
Please change the Id and tenant Id example values. Just to avoid any confusion for the readers